### ISSUE TITLE
Serving Malware

### ISSUE THREAD

**Issue by andrerfneves** [carsenk/denarius#195]
Serving malware. Check https://twitter.com/MisterCh0c/status/1092713969175928834

**Comment by carsenk** [comment]
Investigating this issue currently. Seems my GitHub account was compromised, just secured and investigating further.

**Comment by cmelion** [comment]
@carsenk did you have 2FA enabled?  NewYorkCoin was compromised as a consequence of having a larger Organization that our repo was part of and not making 2FA mandadory.  Contributors to other repos in the org did not have 2FA enabled and were able to silently access release assets.  The only way to detect the intrusion (at least at the time) was to monitor via the api ( https://api.github.com/repos/carsenk/denarius/releases for example).

**Comment by carsenk** [comment]
@cmelion I did not have 2FA enabled at all. They must have gotten Github access and then swapped the binaries. I also had an older password setup on my account previously. So I have it enabled now and changed passwords, cleaned machines, and recompiled a new Windows binary, I ran a scan on it as well as several other InfoSec pros did, it is clean now. I just made this announcement via our Discord:

Update: My Github Account is now 100% secured. The old infected/compromised binaries were removed from the Github. A new v3.3.6 Windows Binary was posted in our releases with SHA256, SHA-1, and MD5 hashes to validate after downloading, ensure that these hashes match after downloading the Denarius.exe to ensure it is clean.  This new binary was validated by multiple InfoSec professionals to be clean.

Release Notes and Binary Hashes: <https://github.com/carsenk/denarius/releases/tag/v3.3.6>
Direct Download Link: <https://github.com/carsenk/denarius/releases/download/v3.3.6/Denarius-v3.3.6-Win64.zip>

The Github source code and repository itself were not affected by this issue so any denariusd nodes running are clean and safe.

~Denarius.exe Windows Binary Hashes~
```
SHA256   3888badd7498c9c6b26825fbf041415def4ae8597207963c38d40f5cd01b7895
MD5    b02a43a707e934143d9ba7a903d44458
SHA-1    7ec9d647fa7a72cb84615048b9660bd1279dc4bd
Authentihash    f9b4b5182df8c06e4c54f7eb6adf5e8c2b7398cbec3da0483a579191147306c7
Imphash    2852de0dcb5e0559e5231e10136fad4b
```
~Denarius-v3.3.6-Win64.zip Hashes~
```
SHA256   b9c721b7c7feb5a8e7f0ecf030804f1715e37c8bcaa91887be3986ddb0a1bc31
MD5    8b27223883ca0b0a583f094b38fb448a
SHA-1    60608f97d1fc3283e84ddf4ed456148bc968e20f
```
